src/my_project/tagging/validator.py

```
@@ -238,167 +199,104 @@ def fuzzy_match(a: str, b: str, threshold: int = 65) -> bool:
+            r"^#\d+$",  # matches "#1", "#1 2" etc
+            r"^issue\s*\d+$",  # matches "issue 3"
+            r"\bvol(?:ume)?\.?\s*(one|two|three|four|\d+|i{1,3}|iv|v)\b",
+            r"\bbook\s*(one|two|three|four|\d+|i{1,3}|iv|v)\b",
+        ]
+        if name:
+            lowered_name = name.lower().strip()
+            if lowered_name in ambig_names or any(
+                re.match(p, lowered_name) for p in ambig_regexes
+            ):
+                if not volume_name:
+                    raise ValueError("No name for issue found.")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)